### DISCUSSION TITLE
erreur index when convert file.mp4 to 1 file.jpg

### DISCUSSION THREAD

**Discussion by D550527V** [ImageMagick/ImageMagick#2859] [Help]
hello
when i convert an mp4 file to obtain 1 phto.jpg, i have an error od index

root@test-normal:/home/admdv# convert /home/admdv/157668727799cd8cf00cab94c64ff4756ea5f78d90.mp4[1000] /home/admdv/test.jpeg
convert-im6.q16: InvalidImageIndex `/tmp/magick-1584Lnoa8FyGJG0X.pam' @ error/list.c/CloneImages/281.
convert-im6.q16: subimage specification returns no images `/tmp/magick-1584Lnoa8FyGJG0X.pam' @ error/constitute.c/ReadImage/642.
convert-im6.q16: InvalidImageIndex `/home/admdv/157668727799cd8cf00cab94c64ff4756ea5f78d90.mp4' @ error/list.c/CloneImages/281.
convert-im6.q16: subimage specification returns no images `/home/admdv/157668727799cd8cf00cab94c64ff4756ea5f78d90.mp4' @ error/constitute.c/ReadImage/642.

Any body have solution ?
Thank You
5i am french)
DV

**Comment by dlemstra** [comment]
What is the output of `magick -version` and where can we find that file? Maybe it has less than 1000 frames?

**Comment by D550527V** [comment]
Hello

The output : 

 

root@test-normal:/var/www/html/piwigo/_data/logs# convert -version

Version: ImageMagick 6.9.10-23 Q16 x86_64 20190101 https://imagemagick.org

Copyright: © 1999-2019 ImageMagick Studio LLC

License:  <https://imagemagick.org/script/license.php> https://imagemagick.org/script/license.php

Features: Cipher DPC Modules OpenMP 

Delegates (built-in): bzlib djvu fftw fontconfig freetype jbig jng jpeg lcms lqr ltdl lzma openexr pangocairo png tiff webp wmf x xml zlib

 

Fic Mp4 :



 

I think 25 * 35 =  875

 

In fact, I want to extract in a picture file the first picture in the film mp4

Thank for your help

 

De : Dirk Lemstra <notifications@github.com> 
Envoyé : mardi 24 novembre 2020 17:01
À : ImageMagick/ImageMagick <ImageMagick@noreply.github.com>
Cc : D550527V <D550527V@mailo.com>; Author <author@noreply.github.com>
Objet : Re: [ImageMagick/ImageMagick] erreur index when convert file.mp4 to 1 file.jpg (#2859)

 

What is the output of magick -version and where can we find that file? Maybe it has less than 1000 frames?
